books-call-number/triggers/startTrigger.json

```
@@ -0,0 +1,9 @@
+{
+  "id": "5667fdf5-fb58-4680-8091-60b64a450fb6",
+  "name": "FOLIO Create Book Report Workflow Trigger",
```

```suggestion
  "name": "Books Call Number Start Trigger",
```
Make the start trigger naming structure consistent with the rest of the workflows.

see: https://github.com/search?q=repo%3ATAMULib%2Ffw-registry+trigger&type=code

It seems that [Create Notes](https://github.com/TAMULib/fw-registry/blob/608d701f27ce511e24bd20e6883cdd21a35fe179/create-notes/triggers/startTrigger.json#L3) is the exception to the pattern and may need to be updated (but not updated here).

Alternatively, if setting this to have `FOLIO` in the name and `Workflow` instead of `Start` shows up differently

---

Start trigger naming structure modified as per suggestion.